ansible/tasks/security/main.yml

```diff
@@ -176,90 +215,67 @@
-    lineinfile:
-      path: /etc/sysctl.conf
-      line: "vm.swappiness = 60"
-  - name: Use Controlled Delay to avoid bufferbloat
-    lineinfile:
-      path: /etc/sysctl.conf
-      line: "net.core.default_qdisc = fq_codel"
-  - name: Deny use of perf_event_open to unprivileged processes
-    lineinfile:
-      path: /etc/sysctl.conf
-      line: "kernel.perf_event_paranoid = 2"
-
-
-
-- name: ssh security settings
-  block:
-  - name: Don't allow Roaming for ssh clients
-    lineinfile:
-      path: /etc/ssh/ssh_config
-      line: "UseRoaming no"
-  - name: Don't use dsa keys in sshd_config
-    lineinfile:
-      path: /etc/ssh/sshd_config
-      line: "HostKey /etc/ssh/ssh_host_dsa_key"
-      state: absent
-  - name: Use ssh privilege separation sandboxing
-    lineinfile:
-      path: /etc/ssh/sshd_config
-      line: "UsePrivilegeSeparation sandbox"
-  - name: Disable password authentication
-    lineinfile:
-      path: /etc/ssh/sshd_config
-      line: "PasswordAuthentication no"
-  - name: Allow use of client-specified gateway ports
-    lineinfile:
-      path: /etc/ssh/sshd_config
-      line: "GatewayPorts clientspecified"
-  - name: Allow root ssh via hardcoded keys as failsafe
-    blockinfile:
-      path: /etc/ssh/sshd_config
-      marker: "# {mark} ANSIBLE - Root SSH via hardcoded keys"
-      block: |
-        Match User root
-             AuthorizedKeysFile        .ssh/authorized_keys
-             AuthorizedKeysCommand     /bin/true
-        Match all
-  - name: Add unbound hardening configuration
-    copy:
-      dest: /etc/unbound/unbound.conf.d/harden.conf
-      content: |
-        server:
-          # Do not expose information about the running deamon
-          hide-identity: yes
-          hide-version:  yes
-
-          # Harden against ridiculously-short buffer sizes (potential DoS vector)
-          # This is against spec, but we aren't a public resolver.
-          harden-short-bufsize: yes
-
-          # Harden against abnormaly large queries (same reasoning)
-          harden-large-queries: yes
-
-          # Return NXDOMAIN for queries under a terminal known (and DNSSEC-validated)
-          #   to be NXDOMAIN.  Improves caching and avoids certain attacks
-          harden-below-nxdomain: yes
-
-          # Use 0x20-encoded random nonces for authenticating queries.
-          # Implementation of draft-dns-0x20, makes DNS poisoning harder
-          use-caps-for-id: yes
-
-          # Minimises queries sent upstream
-          # Avoids information disclosure to root/TLD DNS servers & improves caching
-          qname-minimisation: yes
+- name: "sysctl:  Restrict potentially sensitive kernel dmesg output"
+  lineinfile:
+    create: true
+    path: /etc/sysctl.d/10-dmsg.conf
+    line: "kernel.dmesg_restrict = 1"
+
+- name: "sysctl: Edit sysctl.conf"
+  lineinfile:
+    path: /etc/sysctl.conf
+    line: "{{ item.key }} = {{ item.value }}"
+    regexp: "^#?{{ item.key }} ="
```

Comment:
does not match the old line (see left), make whitespace optional?

Reply:
See left?

Reply:
It doesn't show in the in-conversation view. On the full-diff review, the left side has a line similar to `key=value` - note the lack of space - which your regexp would not match.

Reply:
Oh, yes.